src/Relation/Nullary.agda

```diff
@@ -15,20 +15,9 @@ open import Agda.Builtin.Equality
 ------------------------------------------------------------------------
 -- Re-exports
 
-open import Relation.Nullary.Negation.Core public using
-  ( ¬_; _¬-⊎_
-  ; contradiction; contradiction₂; contraposition
-  )
-
-open import Relation.Nullary.Reflects public using
-  ( Reflects; ofʸ; ofⁿ
-  ; _×-reflects_; _⊎-reflects_; _→-reflects_
-  )
-
-open import Relation.Nullary.Decidable.Core public using
-  ( Dec; does; proof; yes; no; _because_; recompute
-  ; ¬?; _×-dec_; _⊎-dec_; _→-dec_
-  )
+open import Relation.Nullary.Negation.Core public
+open import Relation.Nullary.Reflects public
```

Comment:
Fine. Working on #2149 encouraged me to think about subsequent tightening of these export lists (esp. concerning `Reflects`), but that can follow later.

Reply:
I'm not sure we don't want to tighten them. I think in general we just want to export everything via `Relation.Nullary`. Otherwise, we give ourselves a massive maintenance headache of remembering to update `Relation.Nullary` every time we update one of the submodules?

Reply:
Makes sense, modulo the need to hide certain things (as with the failing test). Again I personally find the trade off between narrowing imports via `using` (a kind of closed world assumption) vs maximally permissive via `hiding` (open world) again, ... a tricky negotiation. For this particular corner of the library, given how much change there has already been to the various representations, exercising some control over visibility (esp. of constructors/patterns, which shouldn't necessarily be exposed) seems a good idea. But happy to be open world for now!

Reply:
> Again I personally find the trade off between narrowing imports via using (a kind of closed world assumption) vs maximally permissive via hiding (open world) again, ... a tricky negotiation. 

I think my mindset is that there are two ways to publicly have module A re-export from module B. Either a) it exports a small fixed closed set of names which are not envisaged to grow as new things are added to B, or b) module B should be designed so that module A should be able to re-export all of it. Anything else is asking for huge maintenance problems in my opinion.

Reply:
You make a convincing case. In this instance, "module B" (`Relation.Nullary.Decidable.Core`) has been a fast-moving target, suggesting option b) is to be preferred.